src/functionalTest/java/submitBulk/ClaimStatusUpdate.java

```diff
@@ -0,0 +1,65 @@
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import io.restassured.path.xml.XmlPath;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.matchesRegex;
+import static org.hamcrest.CoreMatchers.containsString;
+import org.junit.jupiter.api.Assertions;
+import steps.ClaimStatusSteps;
+
+
+@RunWith(SerenityRunner.class)
+public class ClaimStatusUpdate {
+
+    @Steps
+    ClaimStatusSteps claimStatusSteps;
+
+    @Before
+    public void setUp() {
+    claimStatusSteps.setXml(claimStatusSteps.soapRequestBody());
+    claimStatusSteps.setResponse(claimStatusSteps.soapResponse(claimStatusSteps.getXml()));
+}
+
+    @Test
+    public void claimStatusUpdateResponseCode() {
+        XmlPath xmlPath = new XmlPath(claimStatusSteps.getCleanResponse()).setRoot("Envelope.Body.bulkResponse");
```

Comment:
Create a constant for `Envelope.Body.bulkResponse` and use that instead.  As it's also used in the MCOLJudgment and IssueWarrant classes you may want to create it as a public constant on a new parent class and have the three classes inherit from it.